firstvoices/backend/views/custom_order_recalculate_views.py

```diff
@@ -54,3 +63,55 @@ def create(self, request, *args, **kwargs):
 
         except recalculate_custom_order_preview.OperationalError:
             raise CeleryError()
+
+
+class CustomOrderRecalculateView(APIView):
```

Comment:
As mentioned this view works the same as the first iteration of the preview view. I am open to refactoring it if the team would like this to be more consistent with our other APIs. However, I'd need a bit of advice for how to do so as there is no model associated with this API.